packages/teams-js/src/internal/communication.ts

```diff
@@ -518,9 +520,12 @@ function sendMessageToParentHelper(
   args: any[] | undefined,
 ): MessageRequestWithRequiredProperties {
   const logger = sendMessageToParentHelperLogger;
-
   const targetWindow = Communication.parentWindow;
   const request = createMessageRequest(apiVersionTag, actionName, args);
+  HostToAppMessageDelayTelemetry.storeCallbackInformation(request.uuid, {
+    name: actionName,
+    calledAt: request.timestamp,
```

Comment:
@juanscr isnt this supposed to be monotonicTimestamp

Reply:
Hey @sumathin-lang, not necessarily. This information is merely contextual around when the callback happened and isn't used for any calculation. I selected the date instead of the `monotonicTimestamp` as it should be more proper for a global timestamp as it does not possess issues as not ticking while sleeping and so forth (https://github.com/w3c/hr-time/issues/115#issuecomment-1172985601). I can change it to `monotonicTimestamp` if you want but it's a minute detail.

Reply:
@juanscr I would suggest using the same "units" for messageDelay and calledAt. Even if the App wants to use this "calledat" to measure some kind of delta,  perf.now is a better choice compared to date,now. But i will let the teams-js team make this call